app/components/extras.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, final
+
+import discord as dc
+from discord.ext import commands
+
+from app.utils import is_dm, is_mod, try_dm
+
+if TYPE_CHECKING:
+    from app.bot import GhosttyBot
+
+
+@final
+class Extras(commands.Cog):
+    def __init__(self, bot: GhosttyBot) -> None:
+        self.bot = bot
+
+    @commands.command(name="sync", description="Sync command tree.")
+    @commands.guild_only()
+    async def sync(self, ctx: commands.Context[Any]) -> None:
+        assert not is_dm(ctx.author)
+        if not is_mod(ctx.author):
+            return
+
+        await self.bot.tree.sync()
+        await try_dm(ctx.author, "Command tree synced.")
+
+    @dc.app_commands.command(
+        name="status", description="Export Ghostty Bot Status Message"
```

Comment:
```suggestion
        name="status", description="View Ghostty Bot's status."
```

1. “Export […] Status Message” reads as if you're exporting a status message, not exporting the status as a message.
2. Every other command description is in sentence case, with proper punctuation.
3. Exporting makes me think of pulling data out of somewhere so that you can save it, so I think “view” or something else is a better fit.